Lares/Assets/Mechanics/Inventory/UI/InventoryItemButton.cs

```diff
@@ -0,0 +1,29 @@
+using Lares.Inventory;
+using NUnit.Framework.Interfaces;
+using TMPro;
+using UnityEngine;
+using UnityEngine.EventSystems;
+
+namespace Lares.Inventory
+{
+    public class InventoryItemButton : MonoBehaviour, IPointerEnterHandler
+    {
+        public InventoryNode ItemData;
+        [SerializeField] private TextMeshProUGUI _itemName;
+        [SerializeField] private TextMeshProUGUI _numberText;
+
+        public static System.Action<InventoryNode> ItemSelected;
+
+        void Start()
+        {
+            if (ItemData is null) { Debug.Log("item data is null!"); }
+            _itemName.text = ItemData.Item.ItemName;
```

Comment:
```suggestion
            _itemName.text = ItemData.Item.Name;
```
Also will need to rename `ItemData` to chosen name.